src/main/java/org/fundacionjala/coding/SpinWord.java

```diff
@@ -0,0 +1,63 @@
+package org.fundacionjala.coding;
+
+import java.util.StringJoiner;
+
+/**
+ * Created by MarcandeaCabrera on 5/19/2017.
+ */
+
+public class SpinWord {
+    private String cadena;
+    private String[] auxiliarWord;
+    private StringJoiner stringJoiner;
```

Comment:
Do we need this attributes in the class? or Can you handle them in as attribute of method?